tests/modules/cluster.c

```diff
@@ -34,13 +34,52 @@ int test_cluster_shards(ValkeyModuleCtx *ctx, ValkeyModuleString **argv, int arg
     return VALKEYMODULE_OK;
 }
 
+#define MSGTYPE_PING 1
+#define MSGTYPE_PONG 2
+
+/* test.pingall */
+int PingallCommand_ValkeyCommand(ValkeyModuleCtx *ctx, ValkeyModuleString **argv, int argc) {
+    VALKEYMODULE_NOT_USED(argv);
+    VALKEYMODULE_NOT_USED(argc);
+
+    ValkeyModule_SendClusterMessage(ctx, NULL, MSGTYPE_PING, "Hey", 3);
+    return ValkeyModule_ReplyWithSimpleString(ctx, "OK");
+}
+
+/* Callback for message MSGTYPE_PING */
+void PingReceiver(ValkeyModuleCtx *ctx,
```

Comment:
I don't see where this function is called. I can't see where we call `ValkeyModule_RegisterClusterMessageReceiver`. Am I missing something?